spec/spec_helper.rb

```diff
@@ -0,0 +1,15 @@
+require 'simplecov'
+SimpleCov.start
+
+require 'active_support'
+require 'bigdecimal'
+require 'date'
+
+require 'action_controller'
+require 'fixtures/controllers'
+require 'rails_param'
+require 'rails_param/coercion'
+require 'rails_param/param'
+require 'rails_param/validator'
```

Comment:
These files should automatically be required by `rails_param`.
Otherwise, users of the gem will need to manually require them as well